public/javascripts/store.js

```diff
@@ -0,0 +1,39 @@
+//vuex state store to share state between all components
+function makeStore() {
+  var leftPanelClass = 'cc-leftPanel-mobile-hide',
+      userInfoDefault = {
+        email: '',
+        imgSrc: 'https://s.ytimg.com/yts/img/avatar_720-vflYJnzBZ.png'
+      },
+      userInfo = userInfoDefault;
+  if (firebase.auth().currentUser) {
```

Comment:
how about:

`
let user =  firebase.auth().currentUser;

if (user) {
  //.. etc
}
`

This would save you a function call potentially